docs/source/training/socialisation/inasafe_concepts.rst

```diff
@@ -329,7 +331,29 @@ in the globe produced in a standardised and rigorous way.
 +--------------+---------------------------------------------------------------+
 |Notes         | Make sure you know if your data represent density or count    |
 +--------------+---------------------------------------------------------------+
-|Source        | WorldPop or from a  national census agency                    |
+|Source        |Can be obtained from a national mapping agency                 |
++--------------+---------------------------------------------------------------+
+
+Landcover data
+...............
+.. image:: /static/training/socialisation/icon_landcover.png*
```

Comment:
> image:: /static/training/socialisation/icon_landcover.png*

this caused an error. It need to change into:

`image:: /static/training/socialisation/icon_landcover.*
`